src/hpp2plantuml/hpp2plantuml.py

```diff
@@ -988,6 +1011,51 @@ def build_inheritance_list(self):
                             ClassInheritanceRelationship(
                                 parent_obj, obj,
                                 flag_use_namespace=flag_use_namespace))
+                            
+    def build_dependency_list(self):
+        """Build list of dependency between objects
+
+        This method lists all the dependency relationships between objects
+        contained in the `Diagram` object (external relationships are ignored).
+
+        The implementation establishes a list of available classes and loops
+        over objects to obtain their inheritance.  When parent classes are in
+        the list of available classes, their a `ClassInheritanceRelationship`
+        object is added to the list.
+        """
+
+        self._dependency_list = []
+        # Build list of classes in diagram
+        class_list_obj = self._make_class_list()
+        class_list = [c['name'] for c in class_list_obj]
+        flag_use_namespace = any([c['obj']._namespace for c in class_list_obj])
+
+        # Create relationships
+        
+        # Add all objects name to list
+        objects_name = []
+        for obj in self._objects:
+            objects_name.append(obj.get_name())
+
+        # Dependency
+        for obj in self._objects:
+            if isinstance(obj, Class):
+                for member in obj._member_list:
+                    # Check if the member is a method
+                    if isinstance(member, ClassMethod):
+                        for method in member._param_list:
+                            index = ValueError
+                            try:
+                                # Check if the method param type is a Class type
+                                index = objects_name.index(method[0])
+                            except:
+                                pass
+
+                            if index != ValueError and method[0] != obj.get_name():
+                                depend_obj = self._objects[index]
+
+                                self._dependency_list.append(ClassDependencyRelationship(depend_obj, obj, flag_use_namespace=flag_use_namespace))
+            print()
```

Comment:
Can we stay under 79 characters per line?

```suggestion
                            index = ValueError
                            try:
                                # Check if the method param type is a Class
                                # type
                                index = objects_name.index(method[0])
                            except:
                                pass

                            if index != ValueError and \
                               method[0] != obj.get_name():
                                depend_obj = self._objects[index]

                                self._dependency_list.append(
                                    ClassDependencyRelationship(
                                        depend_obj, obj,
                                        flag_use_namespace=flag_use_namespace))
```